packages/client/src/components/GameBoard.tsx

```
@@ -211,6 +212,189 @@ export const GameBoard: React.FC = () => {
     room.send('swapHuzur');
   };
 
+  // ── Drag-and-drop: drop a single card onto the table to play it ──
+  const dropZoneRef = useRef<HTMLDivElement>(null);
```

ESLint

[error] React Hook "useRef" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)